### PR TITLE
Contribute back reindex and tenants rake tasks

### PR DESCRIPTION
**Pull request by summer-cook**
As per slack convo: https://assaydepot.slack.com/archives/C0311DNF3MG/p1687538076547959

### CODE REVIEW

File: lib/tasks/index.rake [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  desc "reindex just the filesets in the background"
+  task reindex_filesets: :environment do
+    Account.find_each do |account|
+      puts "=============== #{account.name}============"
+      next if account.name == "search"
+      switch!(account)
+      title = "~ #{account.name}"
+      progressbar = ProgressBar.create(total: FileSet.count, title: title, format: "%t %c of %C %a %B %p%%")
+      begin
+        FileSet.find_each do |file_set|
```
Comment (ShanaLMoore) [inline lib/tasks/index.rake:52]: You'll need to bring this job over, I don't think it's in hyku.

File: lib/tasks/tenants.rake [hunk 2]
```diff
@@ -0,0 +1,61 @@
+namespace :hyku do
```
Comment (ShanaLMoore) [inline lib/tasks/tenants.rake:1]: Is there a difference between this and the index.rake file?

File: lib/tasks/index.rake [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  task reindex_works: :environment do
+    Account.find_each do |account|
+      puts "=============== #{account.name}============"
+      next if account.name == "search"
+      switch!(account)
+      begin
+        Site.instance.available_works.each do |work_type|
+          title = "~ #{account.name} - #{work_type}"
+          klass = work_type.constantize
+          progressbar = ProgressBar.create(total: klass.count, title: title, format: "%t %c of %C %a %B %p%%")
```
Comment (ShanaLMoore) [inline lib/tasks/index.rake:12]: In Adventist, I was able to run this task locally but wasn't able to in prod without requiring progressbar at the top of this file. 

ref: https://github.com/scientist-softserv/adventist-dl/pull/465/files

Comment (summer-cook) [reply lib/tasks/index.rake:12]: thank you i will look at this!

File: lib/tasks/tenants.rake (hunk 2 above)
Comment (summer-cook) [reply lib/tasks/tenants.rake]: oops, i was doing this way too fast. I just updated it!

**Comment by summer-cook** [comment]
@ShanaLMoore ok - i just updated everything ･ᴗ･ thank you for your review

**Pull request closed without merging**